### PR TITLE
add registry polyfill info to getting started

### PR DESCRIPTION
**Pull request by chrisjbrown**
**This change:** (check at least one)

- [ ] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [ ] Test suite(s) passing?
- [ ] Code coverage maximal?
- [ ] Changeset added?

**What does this change address?**
A clear and concise description or a direct link to an issue [e.g. #15]
If adding a feature, a rationale for its addition goes here.

**How does this change work?**
A clear, detailed description of how this change addresses the issue.
This could be a bullet list if there are several moving parts.

**Additional context**
Add any other context here.

### CODE REVIEW

**Review by jialin-he — APPROVED** [review]

File: packages/pharos-site/src/pages/getting-started.mdx [hunk 1]
```diff
@@ -41,6 +41,13 @@ Once installed, the `node_modules/` directory should contain directories similar
...
 To accomodate patterns of each framework, the implementations vary slightly.
 
+You will first need to provide the scoped custom element registry to your application.
+This is generally achieved by including this script tag in the head of your application.
+
+```javascript
+<script src="https://cdn.jsdelivr.net/npm/@webcomponents/scoped-custom-element-registry@0.0.3/scoped-custom-element-registry.min.js"></script>
+```
+
 You need to first register components on the [custom element registry](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry) for them to be usable in your code.
```
Comment (daneah) [inline packages/pharos-site/src/pages/getting-started.mdx:51]:
```suggestion
You then need to register components on the [custom element registry](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry) for them to be usable in your code.
```

**Review by daneah — COMMENTED** [review]
For continuity

File: packages/pharos-site/src/pages/getting-started.mdx [hunk 2]
```diff
@@ -41,6 +41,13 @@ Once installed, the `node_modules/` directory should contain directories similar
 Pharos uses <PharosLink href="https://developer.mozilla.org/en-US/docs/Web/Web_Components">Web Components</PharosLink> to remain compatible with modern JavaScript frameworks, such as Vue or React.
 To accomodate patterns of each framework, the implementations vary slightly.
 
+You will first need to provide the scoped custom element registry to your application.
+This is generally achieved by including this script tag in the head of your application.
+
+```javascript
+<script src="https://cdn.jsdelivr.net/npm/@webcomponents/scoped-custom-element-registry@0.0.3/scoped-custom-element-registry.min.js"></script>
```
Comment (Niznikr) [inline packages/pharos-site/src/pages/getting-started.mdx:48]: The [latest version is 0.0.5](https://github.com/webcomponents/polyfills/blob/master/packages/scoped-custom-element-registry/package.json#L3). Not sure if that should be the recommendation (perhaps should be tested first).

Comment (Niznikr) [reply packages/pharos-site/src/pages/getting-started.mdx:48]: Use that version on a page that has at least a couple mfes registering components and ensure there aren't any console/JS errors I'd say.

Comment (chrisjbrown) [reply packages/pharos-site/src/pages/getting-started.mdx:48]: updating to version >= 0.0.4 ends up breaking some unit tests
specifically

https://github.com/ithaka/pharos/blob/docs/registry-polyfill/packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.test.ts#L117-L139

**Review by daneah — APPROVED** [review]

**Pull request merged**